bcp/src/app/modules/core-bcp/components/core-consent-modal/core-consent-modal.component.ts

```diff
@@ -27,6 +31,11 @@ export class CoreConsentModalComponent implements AfterViewInit {
   @Output() accept: EventEmitter<boolean> = new EventEmitter<any>();
   @Output() validToken: EventEmitter<string> = new EventEmitter<any>();
 
+  constructor(private spaEnvService: SpaEnvService){
+    const env = this.spaEnvService.getValues();
+    this.recaptcha = env && env.SPA_ENV_ENABLE_RECAPTCHA === 'true';
+  }
```

Comment:
Can you rework the naming of this and use a getter like the address validator toggle does? (instead of doing it in the constructor)

Example:
```typescript
  get isAddressValidatorEnabled(): boolean {
    const env = this.spaEnvService.getValues();
    return env && env.SPA_ENV_ENABLE_ADDRESS_VALIDATOR === 'true';
  }
```

Reply:
Change made.
The only disadvantage of doing it this way is that every time the page renders it will make the request to spa-env-service.  When it is in the constructor it will only make the request once before the first render.  I have made the change to match with the address validator flag for consistancy.

Reply:
Thanks. Yeah, I know about that caveat. Since this is just a web form, we don't really need worry about render performance at this stage since it's a very minute difference. Thanks for pointing that out though!